Add section explaining how to bulk assign tags.

laraaleite opened the pull request:
To bulk assign tags to multiple resources:
1. From any list of resources (e.g. All Resources, Resource Group oveview);
2. Select checkbox for the resources you want to assign the tag;
3. Select Assign tags option;
4. Add name(s) and value(s) and select Assign.

azurecla commented:
Hi __@VinceSmith__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (vincesm@ntdev.microsoft.com). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

azurecla commented:
Hi __@VinceSmith__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (vincesm@ntdev.microsoft.com). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@VinceSmith : Thanks for your contribution to the Azure documentation! The author, @curtand, has been notified to review your proposed change.

curtand commented:
#sign-off

PRmerger commented:
@VinceSmith : Thanks for your contribution to the Azure documentation! The author, @curtand, has been notified to review your proposed change.

laraaleite committed:
Add section explaining how to bulk assign tags. 

To bulk assign tags to multiple resources:
1. From any list of resources (e.g. All Resources, Resource Group oveview);
2. Select checkbox for the resources you want to assign the tag;
3. Select Assign tags option;
4. Add name(s) and value(s) and select Assign.
